### PR TITLE
Only configure stdout logging in setup and teardown hooks

### PR DESCRIPTION
**Pull request by L-MA**
Fix to issue #79 

Also includes bumping libcalico to the correct version that supports specifying rules at profile creation

### CODE REVIEW

File: calico_kubernetes/calico_kubernetes.py [hunk 1]
```diff
@@ -767,6 +767,12 @@ def run():
             logger.info('Executing Calico pod-status hook')
             NetworkPlugin().status(namespace, pod_name, docker_id)
         else:
+            # Append an stdout logging handler to log to the Kubelet
+            # We cannot do this in status hook because K8 looks to stdout
```
Comment (caseydavenport) [inline calico_kubernetes/calico_kubernetes.py]: Spell out Kubelet here rather than abbreviate as "K8"

**Comment by caseydavenport** [comment]
One comment then LGTM.  Make sure it still has 100% coverage.

**Comment by Symmetric** [comment]
@caseydavenport can we make this fix a 1-liner by changing our previous stdout logs to use stderr instead? I'm not crazy about conditionally changing where we make logs, and I suspect that Kubelet will capture stderr in the same way.

**Comment by caseydavenport** [comment]
@Symmetric - Hmm, yeah, that could be a much better way to do this.  We should check that the `kubelet` behaves the way we expect in this case, but if it does, then let's make that change.

**Pull request merged**